.travis.yml

```diff
@@ -15,20 +15,16 @@ matrix:
     - php: 5.3.3
       env: USE_DEPS="lowest"
     - php: 5.6
-      env: SYMFONY_VERSION="2.3.*"
-    - php: 5.6
-      env: SYMFONY_VERSION="2.4.*"
-    - php: 5.6
-      env: SYMFONY_VERSION="2.5.*"
-    - php: 5.6
       env: SYMFONY_VERSION="2.6.*"
     - php: 5.6
       env: SYMFONY_VERSION="2.8.*" PHPUNIT_BRIDGE=true
     - php: 5.6
-      env: SYMFONY_VERSION="dev-master" PHPUNIT_BRIDGE=true MIN_STABILITY="dev"
+      env: SYMFONY_VERSION="3.0.*" PHPUNIT_BRIDGE=true
+    - php: 5.6
+      env: SYMFONY_VERSION="3.1.*@dev" PHPUNIT_BRIDGE=true MIN_STABILITY="dev"
```

Comment:
I actually thought about dropping the stability modification, so I'd like to keep `@dev` just in case.